src/pages/components/carousel.mdx

```diff
@@ -28,6 +23,10 @@ import ResourceLinks from 'components/ResourceLinks';
 
 ## Overview
 
+The Carousel component can be used to display multiple pieces of content in a single viewing area. The Carousel enables users to explore content at their own pace, advancing or returning to previously viewed content. Cards can be used inside the Carousel to condense bite-sized content or act as a jumping off point to explore other pages.
```

Comment:
@kennylam that rings a bell!  I am looking through our website pages and we seem to be pretty inconsistent. Very possible that we decided that and then didn't go through to correct everything 🙁  Looking briefly at the Carbon website they do not capitalize component names.

I do remember deciding to not title case components like Content Block Mixed and use sentence case instead Content block mixed.

Reply:
I think it makes sense to keep `Content block mixed` instead of the lower case version - it would become harder to tell if you're talking about a component.

I can bring in the changes Kenny suggested (single word components are lower case, multi word components are sentence case) and then create a follow-on ticket to make the naming consistent across the site. Sound good @oliviaflory ?